charts/travis-listener/templates/deployment.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "travis-listener.fullname" . }}
+  labels:
+{{ include "travis-listener.labels" . | indent 4 }}
+spec:
+  securityContext:
+    runAsUser: 1000
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "travis-listener.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "travis-listener.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+    {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: {{ .Values.service.port }}
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /uptime
+              port: {{ .Values.service.port }}
+          readinessProbe:
+            httpGet:
+              path: /uptime
+              port: {{ .Values.service.port }}
+          env:
+            {{ range $key, $value := .Values.env }}
```

Comment:
Also here, I'd argue in favor of ConfigMaps instead of using this env list.

Reply:
@Duologic There is alos configmapRef
`envFrom:
            - configMapRef:
                name: {{ template "travis-listener.fullname" . }}`

Reply:
Maybe drop the `env:` then?